synquacer.mk

```diff
@@ -0,0 +1,131 @@
+include common.mk
+include toolchain.mk
+
+################################################################################
+# Paths to git projects and various binaries
+################################################################################
+ARM_TF_PATH ?= $(ROOT)/arm-trusted-firmware
+ARM_TF_OUT ?= $(ARM_TF_PATH)/build/synquacer/$(ARM_TF_BUILD)
+ARM_TF_BL31 ?= $(ARM_TF_OUT)/bl31.bin
+ARM_TF_BL32 ?= $(ARM_TF_OUT)/bl32.bin
+ARM_TF_FIP ?= $(ARM_TF_OUT)/fip_all_arm_tf.bin
+FIPTOOL ?= $(ARM_TF_PATH)/tools/fiptool/fiptool
+EDK2_PATH ?= $(ROOT)/edk2
+EDK2_PLATFORMS_PATH ?= $(ROOT)/edk2-platforms
+EDK2_NON_OSI_PATH ?= $(ROOT)/edk2-non-osi
+EDK2_PKGS_PATH := "$(EDK2_PATH):$(EDK2_PLATFORMS_PATH):$(EDK2_NON_OSI_PATH)"
+EDK2_FIP ?= $(EDK2_NON_OSI_PATH)/Platform/Socionext/DeveloperBox/fip_all_arm_tf.bin
+
+################################################################################
+# Targets
+################################################################################
+.PHONY: all
+all: arm-tf edk2 optee-os
+
+.PHONY: clean
+clean: arm-tf-clean edk2-clean optee-os-clean
+
+################################################################################
+# ARM Trusted Firmware
+################################################################################
+ARM_TF_EXPORTS ?= \
+	CROSS_COMPILE="$(CCACHE)$(AARCH64_CROSS_COMPILE)"
+
+ARM_TF_FLAGS ?= \
+	BL32=$(ARM_TF_BL32) \
+	PRELOADED_BL33_BASE=0x08200000 \
+	DEBUG=$(DEBUG) \
+	PLAT=synquacer \
+	SPD=opteed
+
+.PHONY: arm-tf
+arm-tf: $(ARM_TF_FIP)
+
+.PHONY: arm-tf-clean
+arm-tf-clean:
+	$(ARM_TF_EXPORTS) $(MAKE) -C $(ARM_TF_PATH) $(ARM_TF_FLAGS) clean
+
+$(ARM_TF_FIP): $(ARM_TF_BL32)
+	$(ARM_TF_EXPORTS) $(MAKE) -C $(ARM_TF_PATH) $(ARM_TF_FLAGS) all fiptool
+	$(FIPTOOL) create \
+		--tb-fw $(ARM_TF_BL31) \
+		--soc-fw $(ARM_TF_BL31) \
+		--scp-fw $(ARM_TF_BL31) \
+		--tos-fw $< \
+		$@
+
+$(ARM_TF_BL32): optee-os
+	mkdir -p $(dir $@)
+	$(AARCH64_CROSS_COMPILE)objcopy \
+		-O binary $(OPTEE_OS_PATH)/out/arm/core/tee.elf $@
+
+################################################################################
+# EDK2 / Tianocore
+################################################################################
+define edk2-env
+	export WORKSPACE=$(EDK2_PLATFORMS_PATH)
+endef
+
+define edk2-call
+	GCC5_AARCH64_PREFIX=$(AARCH64_CROSS_COMPILE) \
+	build -n `getconf _NPROCESSORS_ONLN` \
+		-a "AARCH64" -t "GCC5" -b $(EDK2_BUILD) \
+		-p Platform/Socionext/DeveloperBox/DeveloperBox.dsc
+endef
+
+.PHONY: edk2
+edk2: $(EDK2_FIP)
+	$(call edk2-env) && \
+	export PACKAGES_PATH=$(EDK2_PKGS_PATH) && \
+	source $(EDK2_PATH)/edksetup.sh && \
+	$(MAKE) -j1 -C $(EDK2_PATH)/BaseTools && \
+	$(call edk2-call) all
+
+.PHONY: edk2-clean
+edk2-clean: edk2-clean-common
+	cd $(EDK2_NON_OSI_PATH) && \
+		git checkout $(EDK2_FIP)
+
+$(EDK2_FIP): $(ARM_TF_FIP)
+	cp $< $@
+
+################################################################################
```

Comment:
Since there is nothing here, you can just simply remove this Linux part.

Reply:
ok. I will remove the section.